docs/source/deprecation.rst

```diff
@@ -34,7 +34,7 @@ The project offers a comprehensive set of optimized libraries, including the OAP
 DAAL CPP SYCL Interfaces
 ************************
 
-**Deprecation:** The `DAAL CPP SYCL Interfaces <https://github.com/oneapi-src/oneDAL/tree/main/examples/daal/cpp_sycl>`_ in the oneDAL library are deprecated. Starting with the 2024.0 release, oneDAL will not support the DAAL CPP SYCL Interfaces.
+**Deprecation:** The `DAAL CPP SYCL Interfaces in the oneDAL library are deprecated. Starting with the 2024.0 release, oneDAL will not support the DAAL CPP SYCL Interfaces.
```

Comment:
```suggestion
**Deprecation:** The DAAL CPP SYCL Interfaces in the oneDAL library are deprecated. Starting with the 2024.0 release, oneDAL will not support the DAAL CPP SYCL Interfaces.
```

Reply:
Hi @ethanglaser, im not too sure. @maria-Petrova would you be able to confirm?

Reply:
@napetrov DAAL CPP SYCL Interfaces section or all deprecated interfaces section?